src/App.js

```diff
@@ -1,30 +1,40 @@
-import React, { useState, useEffect } from 'react';
-import { Routes, Route } from 'react-router-dom';
-import { getAllData } from './util/index';
-import { HomePage } from './components/HomePage/HomePage';
-import { LoginPage } from './components/LoginPage/LoginPage';
-import ProtectedRoute from './components/ProtectedRoute/ProtectedRoute';
+import React, { useState, useEffect } from "react";
+import { getAllData } from "./util/index";
 
-const URL = 'http://localhost:8000/api/v1/';
 
-const App = () => {
-
-const [message, setMessage] = useState(''); 
+function App() {
+  const [, setMessage] = useState("");
 
   useEffect(() => {
-
     (async () => {
-      const myData = await getAllData(URL)
+      const myData = await getAllData(URL);
       setMessage(myData.data);
     })();
-      
-    return () => {
-      console.log('unmounting');
-    }
 
+    return () => {
+      console.log("unmounting");
+    };
   }, []);
 
   return (
+    <div>
+      <form >
+  
+        <h1>Log in</h1>
+        <label>
+          <p> User Name: </p>
+          <input/>
+        </label>
+        <label>
+          <p> Password</p>
+          <input/>
+          <br />
+        </label>
+        <div>
+          <button type="submit"> Submit </button>
+        </div>
+      </form>
+</div>
     <>
       <h1>{message}</h1>
```

Comment:
This fragment needs to be at the top of the return. We're getting an error right now.